llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -716,6 +825,9 @@ bool RISCVInstructionSelector::select(MachineInstr &MI) {
     return true;
   }
 
+  if (earlySelect(MI, MIB))
```

Comment:
Why does this need to be done "early"? Why can't it be part of the switch after the selectImpl call?